### PR TITLE
lung segmentation ported to ops based FE

### PR DESCRIPTION
**Pull request by rajesh1226**
montgomery.py shall download 138 xray images along with mask and create a csv.
lung_segmentation.py  trains over the xray images to predict masks.

### CODE REVIEW

**Comment by vbvg2008** [comment]
this is a known issue in pipeline, and the fix provided here is not complete because numpy data may still face the same issue. 

I will make sure the issue isn't there I write a clearer version of pipeline.

**Comment by chm123** [comment]
> [CLA assistant check](https://cla-assistant.io/fastestimator/fastestimator?pullRequest=93) 
> Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/fastestimator/fastestimator?pullRequest=93) before we can accept your contribution.
> **1** out of **2** committers have signed the CLA.
> 
> ✅ vbvg2008
> ❌ Ubuntu
> 
> **Ubuntu** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).
> You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/fastestimator/fastestimator?pullRequest=93) it.

@rajesh1226 Please create your account on EC2 and setup the git config user.name and

File: fastestimator/dataset/montgomery.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    df.drop(['basename'], axis=1, inplace=True)
+    df_train = df[:100]
+    df_val = df[100:]
+    df_train.to_csv(train_cvs_path, index=False)
+    df_val.to_csv(eval_cvs_path, index=False)
+
+
+    return train_cvs_path, eval_cvs_path, path 
+
+# path = load_and_set_data()
```
Comment (chm123) [inline fastestimator/dataset/montgomery.py]: Code that is commented out should be deleted.

Comment (vbvg2008) [reply fastestimator/dataset/montgomery.py]: deleted :P

**Pull request closed without merging**